takeaway-challenge.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/dish'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)